frontend/pages/model/[modelId].tsx

```diff
@@ -32,11 +32,21 @@ export default function Model() {
     () =>
       model && uiConfig
         ? [
-            { title: 'Overview', path: 'overview', view: <Overview entry={model} /> },
+            {
+              title: 'Overview',
+              path: 'overview',
+              view: <Overview entry={model} readOnly={!!model.settings.mirror?.sourceModelId} />,
+            },
             {
               title: 'Releases',
               path: 'releases',
-              view: <Releases model={model} currentUserRoles={currentUserRoles} />,
+              view: (
+                <Releases
+                  model={model}
+                  currentUserRoles={currentUserRoles}
+                  readOnly={!!model.settings.mirror?.sourceModelId}
+                />
+              ),
               disabled: !model.card,
               disabledText: 'Select a schema to view this tab',
```

Comment:
This still appears for mirrored models but isn't applicable since a user cannot select a schema